Ian's fix for Cluster Docs (#13897)
ray-project/ray: doc/source/cluster/config.rst

    :ref:`pull_before_run <cluster-configuration-pull-before-run>`: bool
    :ref:`run_options <cluster-configuration-run-options>`:
        - str
    :ref:`disable_automatic_runtime_detection <cluster-configuration-disable-automatic-runtime-detection>`: bool
    :ref:`disable_shm_size_detection <cluster-configuration-disable-shm-size-detection>`: bool
    :ref:`head_image <cluster-configuration-head-image>`: str

>     :ref:`worker_image <cluster-configuration-worker-image>`: str

Reviewer: you added the links without the content:
```
    :ref:`head_image <cluster-configuration-head-image>`: str
    :ref:`worker_image <cluster-configuration-worker-image>`: str
```

> The Docker image to pull in the head and worker nodes. If no image is specified, Ray will not use Docker.

Author: ```suggestion
The default Docker image to pull in the head and worker nodes. This is overridden by the `head_image` and `worker_image` fields. If neither `image` nor (`head_image` and `worker_image`) are specified, Ray will not use Docker.
```

* **Default:** ``[]``
* **Update requires:** Restart

.. _cluster-configuration-disable-automatic-runtime-detection:

``docker.disable_automatic_runtime_detection``
~~~~~~~~~~~~~~~~~~~~~~~~~~

If enabled, Ray will not try to use the NVIDIA Container Runtime if GPUs are present.

* **Required:** No
* **Importance:** Low
* **Type:** Boolean
* **Default:** ``False``

### disable_shm_size_detection
.. _cluster-configuration-disable-shm-size-detection:

``docker.disable_shm_size_detection``
~~~~~~~~~~~~~~~~~~~~~~~~~~

> If enabled, Ray will not automatically specify the size /dev/shm for the started container.

Author: ```suggestion
If enabled, Ray will not automatically specify the size /dev/shm for the started container and the runtime's default value (64MiB for Docker) will be used.
```

* **Default:** ``[]``
* **Update requires:** Restart

.. _cluster-configuration-disable-automatic-runtime-detection:

``docker.disable_automatic_runtime_detection``
~~~~~~~~~~~~~~~~~~~~~~~~~~

> If enabled, Ray will not try to use the NVIDIA runtime if GPUs are present.

Reviewer: ```suggestion
If enabled, Ray will not try to use the NVIDIA Container Runtime if GPUs are present.
```

* **Required:** No

> * **Importance:** Medium

Reviewer: should this be low importance?

Author: yep

> * **Importance:** Medium

Author: ```suggestion
* **Importance:** Low
```

* **Type:** Boolean
* **Default:** ``False``

> * **Update requires:** Restart

Reviewer: please remove this.

> * **Update requires:** Restart

Author: ```suggestion
```

### disable_shm_size_detection
.. _cluster-configuration-disable-shm-size-detection:

``docker.disable_shm_size_detection``
~~~~~~~~~~~~~~~~~~~~~~~~~~

> If enabled, Ray will not automatically specify the size /dev/shm for the started container.

Reviewer: what happens when ray does not automatically specifies it? what the default value will be?

Author: See my suggestion below

* **Required:** No

> * **Importance:** Medium

Reviewer: low importance?

* **Type:** Boolean
* **Default:** ``False``

> * **Update requires:** Restart

Reviewer: please remove this.

> * **Importance:** Medium
> * **Type:** Boolean
> * **Default:** ``False``
> * **Update requires:** Restart

Author: ```suggestion
* **Importance:** Low
* **Type:** Boolean
* **Default:** ``False``
```
